app/controllers/api/v1/sessions_controller.rb

```diff
@@ -0,0 +1,37 @@
+class Api::V1::SessionsController < Api::BaseController
+  def create
+    ensure_one_active_session
+    @session = Session.new(create_params)
```

Comment:
we can compact initialization and persistence in one line by doing:

```suggestion
    @session = Session.create!(create_params)
```